src/ObjCRuntime/ThrowHelper.cs

```diff
@@ -6,10 +6,16 @@
 
 namespace ObjCRuntime {
 
-	static class ThrowHelper {
+	public static class ThrowHelper {
```

Comment:
Maybe hide this class from intellisense?

```suggestion
	[EditorBrowsable (EditorBrowsableState.Never)]
	public static class ThrowHelper {
```